src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HMACSHA512.cs

```diff
@@ -40,11 +40,9 @@ public bool ProduceLegacyHmacValues
             }
             set
             {
-                // We don't have a managed implementation of HMAC
-                // so we can't support this
                 if (value)
                 {
-                    throw new PlatformNotSupportedException();
+                    throw new PlatformNotSupportedException(); // We don't have a managed implementation of HMAC so we can't support this 
```

Comment:
Why would it need to be a managed implementation?  I'm not questioning the PNSE, just the comment explaining the PNSE.